ppa/archive/views.py

```diff
@@ -32,14 +32,30 @@ class DigitizedWorkListView(ListView):
     paginate_by = 50
 
     def get_queryset(self, **kwargs):
-        self.form = SearchForm(self.request.GET)
-        query = join_q = collections = None
+        # Check for a nonsensical search with 'relevance' and no query string
+        # and amend GET dictionary, first removing any blank query strings
+        # so we can just ask if the key exists
+        # testing for empty string specifically to avoid removing integers
+        # for any future query strings
+        GET = {k: v for k, v in self.request.GET.dict().items() if v != ''}
```

Comment:
Pretty sure this is going to cause problems for fields that can be multiple, because I struggled with that in derrida.  See https://github.com/Princeton-CDH/derrida-django/blob/master/derrida/books/views.py#L119-L129

Can we borrow any of the derrida default logic to simplify this?